src/istio/control/http/service_context.h

```diff
@@ -53,6 +53,10 @@ class ServiceContext {
     return service_config_ && !service_config_->disable_report_calls();
   }
 
+  bool ignore_forwarded_attributes() const {
+    return client_context_->IgnoreForwardedAttributes();
```

Comment:
Is it sufficient to scope the setting per listener? Do we have a case when one listener servers both gateway and sidecar destinations, so needs to elect to disable forwarding by service config?